src/main/java/net/oauth/jsontoken/JsonTokenParser.java

```diff
@@ -223,28 +92,13 @@ public boolean issuedAtIsValid(JsonToken jsonToken, Instant now) {
    */
   private List<Verifier> provideVerifiers(JsonToken jsonToken) throws SignatureException {
```

Comment:
Wouldn't it be an issue to change the @throws section in terms of backwards compatibility? Existing code would now be required to wrap usage of the function around a try-catch block for IllegalStateException, when it wasn't necessarily required before (even if it was incorrect behavior).

Reply:
I just meant updating the documentation . Does this method ever throw SignatureException? I can see that it throws IllegalStateException.

Reply:
Ah, I misunderstood. Documentation has been changed to IllegalStateException.

Reply:
It also looks like we can just remove SignatureException since it's not thrown here.
```suggestion
  private List<Verifier> provideVerifiers(JsonToken jsonToken) {
```

Reply:
Removed